### ISSUE TITLE
Cannot find config.h.in

### ISSUE THREAD

**Issue by eroberson** [cole-trapnell-lab/cufflinks#9]
Have the current version of cufflinks from git. No tags, so I'm using master (last commit 815e4350652cf23b817a29e989b261604532781c from Jan 10).

Attempting to build on Ubuntu 12.04.5 LTS, Boost v1.49.0. Eigen 5097c01bcdc4. gcc 4.6.3. Automake v1.11.3.

    ./configure --prefix=/usr/local --with-boost=~/src/boost --with-bam=~/src/samtool_libs --with-eigen=~/src/eigen/Eigen
    
     configure: error: cannot find sources (config.h.in) in . or ..

Any thoughts? Currently have cufflinks v2.1.1 installed, and it had no issues building that version on the same system.

**Comment by kmanalo** [comment]
I'm going to second that- same identical problem... @eroberson is v2.1.1 the last version that worked for you?

**Comment by eroberson** [comment]
It's the last version I *installed*. I'll try to run a systematic check to see where it breaks for me.

Could be related to Boost changes, but I'm not sure how. I would be more likely to believe changes in the Makefile somewhere along the way have caused issue. I still haven't resolved it. I downloaded a 64-bit linux precompiled binary that works, but I don't like not being able to compile it.

**Comment by kmanalo** [comment]
Yeah, It looks like I'm having to follow the same path, I am getting v2.1.1 to install from source, using SLES 11SP3, GCC 4.5.2, Eigen 3.2.4, and SAMTools 0.1.18 (I tried the newer one from htslib.org but read about some compatibility issues).  Thanks for the details, pure coming from a sysadmin point of view on this install.

**Comment by sammyjava** [comment]
Same problem here. I'd be fine with the binary if it supported long Bowtie indexes. But it doesn't, and the current master branch does in src/tophat.py. So I'd really really like to get around this configure issue!
```
sam@ubuntu-12:~/git/tophat$ autoconf
sam@ubuntu-12:~/git/tophat$ ./configure
configure: error: cannot find sources (config.h.in) in . or ..
sam@ubuntu-12:~/git/tophat$ git status
# On branch master
nothing to commit (working directory clean)
sam@ubuntu-12:~/git/tophat$ 
```
